file/Tiltfile

```diff
@@ -0,0 +1,21 @@
+def read_file(path):
+  if os.name == "posix": # Linux/Mac implementation
+    return str(local("cat {}".format(path), echo_off=True, quiet=True))
+  elif os.name == "nt": # Windows implementation
+    path = path.replace("/", "\\")
+    return str(local("type {}".format(path), echo_off=True, quiet=True))
+  else:
+    fail("Unknown system architecture: " + os.name)
+
+def write_file(path, content):
+  content = str(content)
+
+  if os.name == "posix": # Linux/Mac implementation
+    local("echo '{}' > {}".format(content, path), echo_off=True, quiet=True)
```

Comment:
this is going to cause escaping problems, so i'm not sure this is a good idea

Reply:
Np i close it